Machine Learning/sklearn examples/Logistic Regression/Logistic_Regression.ipynb

```diff
@@ -0,0 +1,1438 @@
+{
```

Comment:
<p>Please remove the blank cell at the end</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Rishit-dagli/Python3-by-practice/pull/47/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>

Reply:
Done